README.md

```diff
@@ -76,19 +76,19 @@ Run DB tests by projects will have fewer failures. Here's how you can run them b
 - `sbt resourceValidator/test`
 
 - `sbt janitor/test`
-  
+
   There will be a few failures, but should succeed when run individually. (Currently `com.broadinstitute.dsp.janitor.DbQueryBuilderSpec` has real error that we need to fix at some point).
 
 These are not run in CI, so you have to make sure you run them manually before merging any PRs. Instructions on running these can be found in the respective `DbReaderSpec` files.
 
-2. Once your PR is approved you can merge it and a new PR will be automatically created in [terra-helm](https://github.com/broadinstitute/terra-helm). 
+2. **Merge your `leonardo-cron-jobs` PR.** Once your PR is approved you can merge it and a new PR will be automatically created in [terra-helm](https://github.com/broadinstitute/terra-helm).
 
-3. Get this terra-helm PR merged (you can merge it yourself) and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
+3. **Merge the automatically generated `terra-helm` PR.** You can merge this yourself. Another automatic commit will bump `leonardo`'s chart version. This will trigger another automatic commit
 in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile). Note that this commit will only auto-bump `dev`, and will be auto-merged.
 
-4. Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click the `SYNC APPS` button on the left upper corner. Select these boxes to sync:
+4. **Sync the deployed `leonardo` app in Argo.** Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click the `SYNC APPS` button on the left upper corner. Select these boxes to sync:
```

Comment:
I don't think this is necessary...what happened this time is the bad commit was never merged in `terra-helmfile`

Reply:
These are just headers to make the steps in README more readable. Why not?

Reply:
Or do you mean that step 4 is not necessary and can be removed?

Reply:
I meant there's no need to manually `Sync the deployed `leonardo` app in Argo.`

Reply:
Thanks!